### PR TITLE
Export headers

### PR DESCRIPTION
**Pull request by APalumbo97**

### CODE REVIEW

File: salesforce/my-conservation-life/force-app/main/default/lwc/controllers/__tests__/assetDefinitionsController.test.js [hunk 1]
```diff
@@ -21,6 +21,42 @@ describe('assetDefinitions.findAssetTypes', () => {
...
+    let URL = utils.URL + 'assetPropTypes'
+
+    beforeEach(() => {
+        fetch = createMockFetch(EXPECTED_ASSET_PROP_TYPES);
+        global.fetch = fetch;
+    });
+
+    it('finds all the asset properties for a given asset type', async () => {
+        const assetPropTypes = await assetDefinitions.fetchAssetPropTypes(1);
+        expect(fetch.mock.calls[0][0]).toBe(URL);
```
Comment (tilley14) [inline salesforce/my-conservation-life/force-app/main/default/lwc/controllers/__tests__/assetDefinitionsController.test.js]: check if the "1" is in the body

File: salesforce/my-conservation-life/force-app/main/default/lwc/controllers/__tests__/assetDefinitionsController.test.js [hunk 2]
```diff
@@ -21,6 +21,42 @@ describe('assetDefinitions.findAssetTypes', () => {
...
+    let URL = utils.URL + 'assetPropsByTypeID';
+
+    beforeEach(() => {
+        fetch = createMockFetch(EXPECTED_ASSET_PROPERTIES);
+        global.fetch = fetch;
+    });
+
+    it('finds all the asset properties for a given type of asset', async () => {
+        const assetPropsByTypeID = await assetDefinitions.fetchAssetPropsByTypeID(1);
+        expect(fetch.mock.calls[0][0]).toBe(URL);
```
Comment (tilley14) [inline salesforce/my-conservation-life/force-app/main/default/lwc/controllers/__tests__/assetDefinitionsController.test.js]: check if the "1" is in the body. i.e. if the 2nd parameter in the mock fetch call is {assetTypeID: 1}

File: db/api/v1/db/__tests__/assetDefinitions.db.test.js [hunk 3]
```diff
@@ -146,5 +146,42 @@ describe('assetDefinitions.db.storeCSV', () => {
...
+
+        beforeEach(() => {
+            rows = [{}];
+            query = jest.fn(async() => rows);
+            global.dbPool = { query };
+        });
+
+        it('finds all asset types in the database', async () => {
+            const actualRows = await findAssetTypes();
+            expect(actualRows).toEqual(rows);
```
Comment (tilley14) [inline db/api/v1/db/__tests__/assetDefinitions.db.test.js]: You should test if the correct query is being called in your findAssetTypes

File: db/api/v1/db/__tests__/assetDefinitions.db.test.js [hunk 4]
```diff
@@ -146,5 +146,42 @@ describe('assetDefinitions.db.storeCSV', () => {
...
+        beforeEach(() => {
+            rows = [{}];
+            query = jest.fn(async() => rows);
+            global.dbPool = { query };
+    
+            assetTypeId = 1;
+        });
+    
+        it('finds all properties associated with the asset type ID given', async () => {
+            const actualRows = await findAssetPropTypes(assetTypeId);
```
Comment (tilley14) [inline db/api/v1/db/__tests__/assetDefinitions.db.test.js]: Test if the correct query string and params are being passed to the query function

File: db/api/v1/controllers/__tests__/assetDefinitions.controller.test.js [hunk 5]
```diff
@@ -27,6 +27,62 @@ describe('assetDefinitions.controller.getAssetTypes', () => {
...
+
+        next = jest.fn();
+        expectedAssetTypes = [{}];
+        data = { rows: expectedAssetTypes };
+        assetDefinitionsDb.findAssetPropsByTypeID = jest.fn(async () => data);
+    });
+
+    it('returns all asset types in DB', async () => {
+        await getAssetPropsByTypeID(req, res, next);
+        expect(res.json).toHaveBeenCalledWith(data);
```
Comment (tilley14) [inline db/api/v1/controllers/__tests__/assetDefinitions.controller.test.js]: Should be testing if the controller pulls the correct parameters out of the req and passes it to the db's getAssetPropsByTypeId

File: salesforce/my-conservation-life/force-app/main/default/lwc/exportHeaders/exportHeaders.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<template>
+    <lightning:layout horizontalalign="space">
+        <lightning:layoutItem size="10">
+            <lightning-combobox
+                name="datasource"
```
Comment (exw4141) [inline salesforce/my-conservation-life/force-app/main/default/lwc/exportHeaders/exportHeaders.html]: Could have a better name but not really important

File: db/api/v1/controllers/__tests__/assetDefinitions.controller.test.js (hunk 5 above)
Comment (exw4141) [reply db/api/v1/controllers/__tests__/assetDefinitions.controller.test.js]: @APalumbo97 Is this resolved?

Comment (APalumbo97) [reply db/api/v1/controllers/__tests__/assetDefinitions.controller.test.js]: @exw4141 Not worrying about the tests right now

**Pull request merged**